src/axes/numericAxis.ts

```diff
@@ -84,11 +84,7 @@ export class Numeric extends Axis<number> {
     let domain = scale.domain();
     let min = domain[0] <= domain[1] ? domain[0] : domain[1];
     let max = domain[0] >= domain[1] ? domain[0] : domain[1];
-    if (min === domain[0]) {
-      return scale.ticks().filter((i: number) => i >= min && i <= max);
-    } else {
-      return scale.ticks().filter((i: number) => i >= min && i <= max).reverse();
```

Comment:
Yeah the .reverse() was only to simplify the `_hasOverlapWithInterval` method, which I've refactored to be direction agnostic